app/views/claims/eligibility_confirmed.html.erb

```diff
@@ -6,6 +6,6 @@
     <h1 class="govuk-heading-xl">You are eligible to claim back student loan repayments</h1>
     <p class="govuk-body">Based on your answers, you can claim back the student loan repayments you made between 6 April 2018 and 5 April 2019.</p>
 
-    <%= link_to "Continue", claim_path("information-provided"), class: "govuk-button", role: "button" %>
+    <%= link_to "Continue", claim_path("student-loan-amount"), class: "govuk-button", role: "button" %>
```

Comment:
could we make this less brittle to changes in order by using the page sequence somehow to determine what this link is rather than hard-coding the slug?